base/stream.jl

```diff
@@ -433,30 +412,6 @@ end
 
 alloc_buf_hook(stream::LibuvStream, size::UInt) = alloc_request(stream.buffer, UInt(size))
 
-function notify_filled(buffer::IOBuffer, nread::Int, base::Ptr{Void}, len::UInt)
```

Comment:
that can be put back then. looks like it might be good to merge some parts of https://github.com/JuliaLang/julia/pull/4484/files too to document it